### ISSUE TITLE
Add some motivational quotes in the browser ?

### ISSUE THREAD

**Issue by Niyant1** [PrakharPythonProgrammer/Grail-0.6#14]
Hi Everyone
Good Evening PPP Originator.... I came up with idea that we can put motivational quotes while in free time they can have motivation...and we will update these quotes every time whenever he/she opens it.... 
So I hope u liked my idea...

**Comment by PrakharPythonProgrammer** [comment]
Yes your idea is good but from next time no need to mention good evening
originator and all that. And others what are you opinions about this idea ?
